best_practices/deploy/deploy

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+#############MANDATORY#########
+PROJECT_NAME=
+###############################
+
+if [ -z "$DOCKERHUB_USER" ]; then
+  DOCKERHUB_USER='codelittinc'
+fi
+
+if [ -z "$DEPLOY_TYPE" ]; then
+  exit "You need to pass the deploy type(DEPLOY_TYPE), example: qa, production, beta e etc."
+fi
+
+if [ -z "$SERVER_USER" ]; then
+  exit "You need to pass the server user(SERVER_USER) of the application will be running."
```

Comment:
Grammar correction: 

`exit "You need to pass the server username (SERVER_USER) for the deployment of this application."`